types/api/v1beta1/event.proto

```diff
@@ -50,6 +50,7 @@ message Parent  {
 
 message Executable {
     string name = 1;
```

Comment:
I wonder if we should make name optional as well since it can be computed from path (basename), wdyt?

Reply:
Waiting on resolution for this one to merge (or not)